lib/logstash/plugin_mixins/jdbc.rb

```diff
@@ -205,26 +207,29 @@ def execute_statement(statement, parameters)
       query = @database[statement, parameters]
       sql_last_value = @use_column_value ? @sql_last_value : Time.now.utc
       @tracking_column_warning_sent = false
-      @logger.debug? and @logger.debug("Executing JDBC query", :statement => statement, :parameters => parameters, :count => query.count)
```

Comment:
Why was `:count` removed? Did it execute an extra `COUNT` or was there some other reason?